conans/client/conan_api.py

```diff
@@ -1100,13 +1098,42 @@ def read_profile(self, profile=None):
 
     @api_method
     def get_path(self, reference, package_id=None, path=None, remote_name=None):
+
+        def get_path(cache, ref, path, package_id):
+            """ Return the contents for the given `path` inside current layout, it can
+                be a single file or the list of files in a directory
+
+                :param package_id: will retrieve the contents from the package directory
+                :param path: path relative to the cache reference or package folder
+            """
+            assert not os.path.isabs(path)
+
+            if package_id is None:  # Get the file in the exported files
+                latest_rrev = cache.get_latest_rrev(ref)
+                folder = cache.get_ref_layout(latest_rrev).export()
+            else:
+                latest_pref = cache.get_latest_prev(PackageReference(ref, package_id))
+                folder = cache.get_pkg_layout(latest_pref).package()
```

Comment:
This will fail because you don't have the recipe revision for the reference. It's passing in the PR because `ConanGetTest` are marked as xfail